src/VisualStudio/Core/Def/Implementation/ProjectSystem/AbstractEncProject_IVsENCRebuildableProjectCfg.cs

```diff
@@ -20,57 +19,58 @@ public int HasCustomMetadataEmitter(out bool value)
 
         public int StartDebuggingPE()
         {
-            return EditAndContinueImplOpt.StartDebuggingPE();
+            return EditAndContinueImplOpt != null ? EditAndContinueImplOpt.StartDebuggingPE() : VSConstants.S_OK;
```

Comment:
Can be written as

    return EditAndContinue?.StartDebuggingPE() ?? VSConstants.S_OK;